bin.src/createDetGeom.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
+#createRefSpec.py '/Users/azuri/spectra/pfs/PFS' --id visit=4 filter='r' spectrograph=2 site='F' category='A' -C ../obs_pfs/config/pfs/createRefSpec.py --output "/Users/azuri/spectra/pfs/PFS" --clobber-config
+from pfs.drp.stella.createDetGeom import CreateDetGeomTask
+task = CreateDetGeomTask()
+task.run()
```

Comment:
Added a comment to bin.src/createDetGeom.py stating that CreateDetGeomTask is a Task and not a CmdLineTask